### ISSUE TITLE
Revise settings.yaml

### ISSUE THREAD

**Issue by ktfhale** [STScI-Citizen-Science/MTPipeline#130]
`settings.yaml` should be edited to only have the variables we actually want to use, and the rest of the pipeline should be edited to actually use those variables.

We should have `run_imaging_pipeline` use the number of cores specified in the `settings.yaml` file. At the moment, it's hardcoded to use the total number of available cores, minus 1. At the moment, I've hardcoded the number of cores to 4 in master, for ease in running the pipeline.

We should delete the fits file input/output paths for the various instruments. Inputs are currently set by the `-filelist` input argument when calling `run_imaging_pipeline` from the terminal, and the pipeline distinguishes instruments using header data anyway. While I don't know exactly how it works, pipeline outputs always seem to go in the same directory as their respective input `_c0m` or `_flt` file. This may not be desirable behavior for the png outputs.

We'll keep the database and email flags, as well as the logging path. But I've noticed that actually running the pipeline involves creating a `run_imaging_pipeline/` directory inside whatever folder you specify as the logging directory. I'll fix that too.

**Comment by ktfhale** [comment]
I've updated `run_imaging_pipeline()` to use the `num_cores` entry from the `SETTINGS` dictionary. I've also cleaned up the `template_settings.yaml` file, although I've kept the old one around for now as `old_template_settings.yaml`. Finally, I've gotten rid of `module_name` from the logging path, so that the pipeline's logs actually go into the path specified in `settings.yaml`.

If we do implement logging for the other modules, we could put the module name back in the path. But at the moment, the pipeline fails to run if the user doesn't beforehand manually create a `run_imaging_pipeline` directory inside the directory specified by `logging_path`, which they had no way of knowing they needed to do. If we want to add module-specific logging, we should at least have the pipeline automatically create those folders.

I think this branch is ready to be merged.
